docs/changelogs/CHANGELOG-2.23.md

```diff
@@ -15,6 +15,13 @@ Before upgrading, make sure to read the [general upgrade guidelines](https://doc
 - Introduce `EnableShareCluster` flag in `KubermaticSettings` to toggle the share cluster feature for the dashboard ([#11950](https://github.com/kubermatic/kubermatic/pull/11950))
   - **ACTION REQUIRED:** `share_kubeconfig` field in the UI configuration for KubermaticConfiguration has been replaced with `EnableShareCluster` flag in KubermaticSettings. `share_kubeconfig` is no-op and will be ignored by the dashboard
 
+### Known Issues
```

Comment:
Can we add a little explanation under this headline, saying something like "the following issues have been found in this release and will be fixed in upcoming patch releases"? Otherwise one could understand these are "known issues that were fixed in this release".

Reply:
good point, added